src/redshift-adjust

```diff
@@ -42,10 +42,14 @@ EOF
 fi
 
 if [ ! -e "${file_temp}" ]; then
-    echo 6500 > "${file_temp}"
+    temperature=$(redshift -p | grep -Pzo "(?<=Color temperature: ).*")
+    temperature=${temperature%?} # remove last char (K)
```

Comment:
This should be combined with `sed` above, and the `K` should be explicit.

Reply:
Here, do you mean that I should remove the K explicitly by using, e.g.,
```shell-script
redshift -p | grep 'Color temperature:' | sed -e 's/Color temperature: \(.*\)K/\1/'
```
or that I should leave the K in the file?